mobility/framestore/tools/global_transforms.cc

```diff
@@ -69,7 +65,7 @@ int main(int argc, char **argv) {
           && msg_conversions::config_read_vector(&t_trans, &trans)) {
           // Only send global transforms
           if (global) {
-            tf.header.stamp = ROS_TIME_NOW();
+            tf.header.stamp = node_->get_clock()->now();
```

Comment:
FF_TIME_NOW?

Reply:
So FF_TIME_NOW() uses the variable clock_ and not node_ :/ not sure what to do about these edge cases, it seemed weird to create clock_ just to call this given that it would be custom ROS2 code anyway, Maybe have a FF_TIME_NOW_NODE and FF_TIME_NOW that uses clock_? What do you think?

Reply:
It's probably not too important, but why make the clock_ variable at all? We could just call get_clock in the macro. I doubt get_clock is an expensive operation?

Reply:
Yeah it was one of the things that I didn't have a preference for and chose randomly whether to store a node_ variable or clock_ variable :)